avogadro/rendering/solid_first_fs.glsl

```diff
@@ -32,14 +32,20 @@ uniform float offset;
 uniform sampler2D inDepthTex;
 // 1.0 if enabled, 0.0 if disabled
 uniform float inAoEnabled;
+// 1.0 if enabled, 0.0 if disabled
+uniform float inDofEnabled;
 // 0.0 if disabled
 uniform float inFogStrength;
 // Shadow strength for SSAO
 uniform float inAoStrength;
 // 1.0 if enabled, 0.0 if disabled
 uniform float inEdStrength;
-// offset for zoom-in and zoom-out
+// amount of offset when zoom-in or zoom-out.
 uniform float uoffset;
+// Dof strength
```

Comment:
Much like the fog strength, I'd much rather move to one float for inDofStrength == 0.0 if disabled. (I'll be submitting a patch for the ambient occlusion in a moment.)